ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10.yaml

```diff
@@ -36,6 +42,21 @@ tests:
     - chain: ipi-aws-pre
     - ref: cucushift-pre
     workflow: cucushift-e2e
+- as: e2e-aws-arm64-ipi
+  interval: 24h
+  steps:
+    cluster_profile: aws-arm64
```

Comment:
Should this be the QE AWS profile @yunjiang29 ?

Reply:
if it's part of QE profiles, the answer is yes.

@aleskandro is there any special credentials required? or just pull secret and AWS credential is enough?

Reply:
@yunjiang29 no special credentials. We use the same AWS accounts for both x86 and arm testing in Jenkins.

Reply:
OK , `aws-qe` is ready now, use it if required.

Reply:
The `aws-qe` won't work with arm64. The quota-slice for each architecture is different. For reference -> https://github.com/openshift/release/blob/master/core-services/prow/02_config/_boskos.yaml#L174-L210 
Available options:
1. Add a quota-slice for `aws-arm64-qe`
2. Use the same quota-slice as of arm64 CI `aws-arm64`